crates/graphql/src/schema/objects/nft.rs

```diff
@@ -145,27 +151,34 @@ impl Nft {
         &self.description
     }
 
-    #[graphql(arguments(width(description = "image width"),))]
+    #[graphql(arguments(width(
+        description = "Image width possible values are:\n- 0 (Original size)\n- 600 (Small)\n- 800 (Medium)\n- 1400 (Large)\n\n Any other value will return the original image size.\n\n If no value is provided, it will return width 800"
+    ),))]
     pub fn image(&self, width: Option<i32>, ctx: &AppContext) -> String {
         let width = ImageSize::from(width.unwrap_or(ImageSize::Medium as i32));
 
+        let mut hasher = DefaultHasher::new();
```

Comment:
This uses random-state hashing, which means it will produce an inconsistent result for every call.  Is this okay?